pkg/features/kube_features.go

```diff
@@ -98,7 +94,6 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
-	ExternalTrafficLocalOnly:                    {Default: true, PreRelease: utilfeature.GA},
```

Comment:
@thockin I put more thoughts on it. It seems safe to remove it from feature gate.

The only cases we may concern about would be:
1. Feature is enabled on master but disabled on nodes (only upgrade master).
2. Feature is disabled on master but enabled on nodes (only upgrade nodes).

For the first case, users disabled this feature before 1.7. Someday their master gets upgraded to 1.7 and there is not way to turn it off on master, which introduces a mismatch between master and nodes. If they don't explicitly create OnlyLocal Services, all components will act as usual. Even if they do mistakenly create OnlyLocal Service, the local traffic healthcheck on LB will fail on all nodes (because feature is disabled on nodes), hence fall back to forwarding traffic to all nodes. Nothing will be broken.

For the second case, there won't be issue unless users **explicitly** create OnlyLocal Service while this feature is disabled on master. In this case LB will be targeting all nodes, while some of the nodes will drop traffic if there is no endpoints on it.